packages/form/tests/integration/components/cf-field/input/checkbox-test.js

```diff
@@ -26,6 +26,11 @@ module("Integration | Component | cf-field/input/checkbox", function (hooks) {
           )
           question=(hash
             __typename="MultipleChoiceQuestion"
+            raw=(hash
+              meta=(hash
+                vertical=false
+              )
+            )
```

Comment:
Why is this needed, isn't it normal that this would be non-existent?

Reply:
Youe are right. This is not needed.